kv-components/vue/stories/KvProgressBar.stories.js

```diff
@@ -0,0 +1,34 @@
+import KvProgressBar from '../KvProgressBar.vue';
+
+export default {
+	title: 'KvProgressBar',
+	component: KvProgressBar,
+	args: {
+		value: 50,
+		max: 100,
+	},
+	argTypes: {
+		value: {
+			control: 'range',
+			options: {
+				min: 0,
+				max: 100,
+			},
+		},
+	},
+};
+
+const Template = (args, { argTypes }) => ({
+	props: Object.keys(argTypes),
+	components: { KvProgressBar },
+	template: `
+		<kv-progress-bar
+			ref="progressBarRef"
```

Comment:
Is this needed?

Reply:
Thanks for catching this Eddie, it's been removed in the latest commit.